Chargebee/Classes/Result/CBError.swift

```diff
@@ -11,6 +11,8 @@ public enum CBError: Error {
     case operationFailed(errorResponse: CBErrorDetail)
     case invalidRequest(errorResponse: CBErrorDetail)
     case paymentFailed(errorResponse: CBErrorDetail)
+    case serverNotResponding(errorResponse: CBErrorDetail)
```

Comment:
Quick thought. Should we rename this to some generic server error like`serverError`?
Reason is that when we say `serverNotResponding` it implies that we dont get any response from the server.

Reply:
updated done